types/ax/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for ax 0.1
+// Project: https://github.com/AstroxNetwork/ax
+// Definitions by: Hsehoiki <https://github.com/AstroxNetwork>
```

Comment:
Owners should be actual people, not organizations

Reply:
I re-edited and commited.